requirements-dev.txt

```diff
@@ -12,6 +12,7 @@ coverage==5.5
 # To run make check
 aexpect==1.6.1
 psutil==5.8.0
+bandit==1.7.2; python_version > '3.6'
```

Comment:
So, this means that when in python3.6 `selftests/bandit-check.sh` it will fail?

Reply:
This means bandit will be only installed in Python 3.7 and superior.
What change are you requesting here?

Reply:
At this point I'm not requesting any changes, @ana. I'm just trying to understand why it is ok to fail this when running on python3.6:

A `make check` and individual run of `selftests/bandit-check.sh` it will fail because bandit it will not be installed.

Reply:
> A `make check` and individual run of `selftests/bandit-check.sh` it will fail because bandit it will not be installed.

Yeah, at this stage I hope no one is using Python 3.6 for developing.

Reply:
Ooops, I did not catch that, but you're right.

The way I see it, we either add another special condition (I don't like special conditions), or have Python 3.6 removed before this is applied.

BTW, John Snow (not tagging him to avoid noise) was considering pushing a strategy for having Python 3.7+ stacks installed in the Docker images that build QEMU on platforms still with Python 3.6.

The other possibility is to declare Avocado 3.7+ already, and protect the `make check-avocado` execution on QEMU against lesser versions.